apps/server/.env.example

```
@@ -56,4 +56,18 @@ JWT_SECRET="your-super-secret-and-long-string"
+# REDIS_URL=redis://localhost:6379
+# 或者使用单独的连接参数（与 REDIS_URL 二选一）
+# REDIS_HOST=localhost
+# REDIS_PORT=6379
+# REDIS_PASSWORD=
+# REDIS_DB=0
+# 缓存默认 TTL，单位：秒（0 表示不过期）
+# REDIS_DEFAULT_TTL=300
+# Key 前缀，用于多实例 / 多环境隔离
+# REDIS_KEY_PREFIX=my-first-nest:
+# 是否强制要求 Redis 可用；为 true 时连接失败将阻塞启动
+# REDIS_REQUIRED=false
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)